interpreter/text/arrange.ml

```diff
@@ -85,9 +85,8 @@ let valtype t = string_of_valtype t
 let storagetype t = string_of_storagetype t
 
 let heaptypeuse = function
```

Comment:
Nit: `heaptypeuse` no longer is an adequate name. `exacttypeuse` like in the parser?